### PR TITLE
RHOARDOC: 1670 Removed modular content for example application assemblies

### PR DESCRIPTION
**Pull request by huntley**
@tradej @inoxx03 I removed the introductory modules as they are no longer used by the launcher and integrated that content into the assemblies. When the assembly already had content intended to be read after the included intro, I moved that content into a new module. There are some minor content updates. The modules I tend to use for the Launcher capabilities will have additional editorial changes but I will make those changes in separate PRs that correspond to each capability.

### CODE REVIEW

File: docs/topics/assembly_crud-example-application.adoc [hunk 1]
```diff
@@ -9,39 +9,15 @@
...
+Example proficiency level:
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://launcher.fabric8.io/docs/thorntail-runtime.html#proficiency_levels[*{proficiency-foundational}*^].]
 
-The example application exposes an HTTP API, which provides endpoints that allow you to manipulate data by performing  _CRUD_ operations over HTTP.
-The _CRUD_ operations are mapped to HTTP `Verbs`.
-The API uses JSON formatting to receive requests and return responses to the user.
-The user can also use a user interface provided by the example to use the application.
-Specifically, this example provides an application that allows you to:
+The {name-example-crud} example expands on the {name-example-http-api} application to provide a basic example of performing _create_, _read_, _update_ and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API. _CRUD_ operations are the four basic functions of persistent storage, widely used when developing an HTTP API dealing with a database.
```
Comment (ncbaratta) [inline docs/topics/assembly_crud-example-application.adoc]:
```suggestion
The {name-example-crud} example expands on the {name-example-http-api} application to provide a basic example of performing _create_, _read_, _update_, and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API. _CRUD_ operations are the four basic functions of persistent storage, widely used when developing an HTTP API dealing with a database.
```

File: docs/topics/con_purpose-of-the-health-check-pattern.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[id='purpose-of-the-health-check-pattern_{context}']
+= Purpose of the Health Check Pattern
```
Comment (ncbaratta) [inline docs/topics/con_purpose-of-the-health-check-pattern.adoc]: Titles should be in sentence case per a style decision from CCS

Comment (ncbaratta) [inline docs/topics/con_purpose-of-the-health-check-pattern.adoc]:
```suggestion
= Health check pattern
```

File: docs/topics/con_purpose-of-the-http-api-pattern.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[id='purpose-of-the-http-api-pattern_{context}']
+= Purpose of the HTTP API Pattern
```
Comment (ncbaratta) [inline docs/topics/con_purpose-of-the-http-api-pattern.adoc]:
```suggestion
= HTTP API pattern
```

File: docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
+[id=purpose-of-the-relational-database-backend-pattern_{context}]
+= Purpose of the Relational Database Backend pattern
```
Comment (ncbaratta) [inline docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc]:
```suggestion
= Relational database backend pattern
```

File: docs/topics/assembly_crud-example-application.adoc (hunk 1 above)
Comment (huntley) [reply docs/topics/assembly_crud-example-application.adoc]: Fixed.

File: docs/topics/con_purpose-of-the-health-check-pattern.adoc (hunk 2 above)
Comment (huntley) [reply docs/topics/con_purpose-of-the-health-check-pattern.adoc]: Fixed throughout.

Comment (huntley) [reply docs/topics/con_purpose-of-the-health-check-pattern.adoc]: Fixed.

File: docs/topics/con_purpose-of-the-http-api-pattern.adoc (hunk 3 above)
Comment (huntley) [reply docs/topics/con_purpose-of-the-http-api-pattern.adoc]: Fixed.

File: docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc (hunk 4 above)
Comment (huntley) [reply docs/topics/con_purpose-of-the-relational-database-backend-pattern.adoc]: Fixed.

File: docs/topics/assembly_configmap-example-application.adoc [hunk 5]
```diff
@@ -10,7 +10,12 @@
 [id='example-configmap-{parameter-runtime-anchor}']
 = {name-example-configmap} example for {runtime}
 
-include::configmap-example-intro-paragraph.adoc[leveloffset=+1]
+Example proficiency level:
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://launcher.fabric8.io/docs/thorntail-runtime.html#proficiency_levels[*{proficiency-foundational}*^].]
+
+
+{name-example-configmap} provides a basic example of using a ConfigMap to externalize configuration. _ConfigMap_ is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
```
Comment (ncbaratta) [inline docs/topics/assembly_configmap-example-application.adoc]:
```suggestion
{name-example-configmap} provides a basic example of using a ConfigMap to externalize configuration. A ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
```

Comment (ncbaratta) [inline docs/topics/assembly_configmap-example-application.adoc]:
```suggestion
{name-example-configmap} provides a basic example of using a ConfigMap to externalize configuration. A ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
```

**Comment by inoxx03** [comment]
[test]

**Comment by inoxx03** [comment]
@huntley Are you still working on this PR? If you're done making changes to it, please accept or reject suggestions provided by @ncbaratta and I will merge this PR.

**Comment by huntley** [comment]
Thanks @inoxx03! I'm done working on this set of changes. There will be more updates to come.

**Comment by inoxx03** [comment]
[test]

**Comment by inoxx03** [comment]
Thank you @huntley, merging...

**Comment by inoxx03** [comment]
@huntley @tradej Re Opening this as a WIP, to be included in the next docs release.

File: docs/topics/assembly_circuit-breaker-example-application.adoc [hunk 6]
```diff
@@ -9,12 +9,16 @@
 [id='example-circuit-breaker-{parameter-runtime-anchor}']
 = {name-example-circuit-breaker} example for {runtime}
 
-include::note-example-unavailable-oso.adoc[leveloffset=+1]
+Example proficiency level: 
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://launcher.fabric8.io/docs/thorntail-runtime.html#proficiency_levels[*{proficiency-foundational}*^].]
 
-include::circuit-breaker-example-intro-paragraph.adoc[leveloffset=+1]
+The _Circuit Breaker_ example demonstrates a generic pattern for reporting the failure of a service and then limiting access to the failed service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
```
Comment (tradej) [inline docs/topics/assembly_circuit-breaker-example-application.adoc]: One sentence per line, pls.

File: docs/topics/con_purpose-of-the-health-check-pattern.adoc [hunk 7]
```diff
@@ -0,0 +1,13 @@
+[id='purpose-of-the-health-check-pattern_{context}']
+= Health check pattern
```
Comment (tradej) [inline docs/topics/con_purpose-of-the-health-check-pattern.adoc]: Module heading doesn't correspond to the file name and the anchor.

File: docs/topics/assembly_circuit-breaker-example-application.adoc [hunk 8]
```diff
@@ -9,12 +9,16 @@
 [id='example-circuit-breaker-{parameter-runtime-anchor}']
 = {name-example-circuit-breaker} example for {runtime}
 
-include::note-example-unavailable-oso.adoc[leveloffset=+1]
+Example proficiency level: 
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
```
Comment (tradej) [inline docs/topics/assembly_circuit-breaker-example-application.adoc]: I believe this is a relic of the reuse in the launcher interface. Please investigate and remove the conditional if it's the case.

Comment (huntley) [reply docs/topics/assembly_circuit-breaker-example-application.adoc]: It appears that this information is still in the booster-catalog (https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/metadata.yaml#L30). The new Launcher UI does not use it but this information is returned by the backend. So if does not appear in the Launcher UI, can it be removed?

Comment (tradej) [reply docs/topics/assembly_circuit-breaker-example-application.adoc]: Please verify with Engineering if this very file is being used. The metadata file does indeed contain a reference to the proficiency level, but this `ifdef` only modifies whether an `xref` or a `link` is used, which doesn't seem related to me.

Comment (huntley) [reply docs/topics/assembly_circuit-breaker-example-application.adoc]: Confirmed that it is not referenced in any Launcher source files by @quintesse. I'll remove the conditional and the xref and keep the link.

**Pull request merged**